src/test/java/scenarios/entity/EntitiesSteps.java

```diff
@@ -58,10 +59,12 @@ public void iAddThisEndpointAndSendTheRequestWithBody(final String endpoint) thr
         logger.info("=================== Create When ==============================");
         logger.info(body);
         apiResponse = ApiRequestManager.create(endpoint, pathParams, body);
-        CreatedResponse createdResponseHelper = apiResponse.getResponse().as(CreatedResponse.class);
-        createdResponse.setId(createdResponseHelper.getId());
-        createdResponse.setSuccess(createdResponseHelper.isSuccess());
-        createdResponse.setErrors(createdResponseHelper.getErrors());
+        if (apiResponse.getStatusCode() == HttpStatus.SC_CREATED) {
+            CreatedResponse createdResponseHelper = apiResponse.getResponse().as(CreatedResponse.class);
+            createdResponse.setId(createdResponseHelper.getId());
+            createdResponse.setSuccess(createdResponseHelper.isSuccess());
+            createdResponse.setErrors(createdResponseHelper.getErrors());
```

Comment:
If the success and error fields are not gonna be used, then it may be better to not add unnecessary values.

Reply:
It's right. I removed them.